docs/articles/features/vsprofiler.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: docs.vsprofiler
```

Comment:
Our smart spell check fails with an error for the words "vsprofiler" and "diagsession".

```log
Run cspell --config ./cSpell.json "docs/**/*.md" --no-progress
./docs/articles/features/vsprofiler.md:2:11 - Unknown word (vsprofiler)
./docs/articles/features/vsprofiler.md:13:382 - Unknown word (diagsession)
./docs/articles/features/vsprofiler.md:13:[4](https://github.com/dotnet/BenchmarkDotNet/actions/runs/12190539263/job/34028617774?pr=2672#step:6:5)43 - Unknown word (diagsession)
./docs/articles/features/vsprofiler.md:69:18[5](https://github.com/dotnet/BenchmarkDotNet/actions/runs/12190539263/job/34028617774?pr=2672#step:6:6) - Unknown word (diagsession)
./docs/articles/samples/IntroVisualStudioProfiler.md:14:49 - Unknown word (diagsession)
```

Could you please add these two words to the `cSpell.json` file?

https://github.com/dotnet/BenchmarkDotNet/blob/5f0c47bfd15700e63f47870742282d417258c988/build/cSpell.json#L14